### ISSUE TITLE
Max height for posts on UserPostsPage

### ISSUE THREAD

**Issue by ghost** [flarum/issue-archive#376]

**Comment by tobyzerner** [comment]
I think I agree that we should give a max-height to posts on the user profile page, with a little expand icon/button down the bottom. Similar to flarum/issue-archive#375.

**Comment by franzliedke** [comment]
Agreed, we'll do the same as in flarum/issue-archive#375, but for all long posts.
